fedimg/utils.py

```diff
@@ -99,13 +101,15 @@ def get_value_from_dict(_dict, *keys):
 
 
 def external_run_command(command):
-    _log.debug("Starting the command: %r" % command)
-    ret = subprocess.Popen(' '.join(command), stdin=subprocess.PIPE, shell=True,
-                           stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-                           close_fds=True)
+    _log.info("Starting the command: %r" % command)
+    ret = subprocess.Popen(' '.join(command), stdin=subprocess.PIPE,
+                           shell=True, stdout=subprocess.PIPE,
+                           stderr=subprocess.PIPE, close_fds=True)
     out, err = ret.communicate()
-    _log.debug("Finished executing the command: %r" % command)
     retcode = ret.returncode
+    _log.info("Finished executing the command: %r" % command)
+    _log.debug("(output) %s, (error) %s, (retcode) %s" % (
+               out, err, retcode))
```

Comment:
i would say this line should get output any time the returncode != 0

Reply:
It would always log the output in `DEBUG` level.

Reply:
i'm thinking it should be
```
if ret.returncode != 0:
     _log.error("command failed during run") 
     _log.error("(output) %s, (error) %s, (retcode) %s" % (out, err, retcode))
```

Reply:
Updated them. Good to merge?